ui-manchette-with-spacetimechart/src/config.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+  const content: string;
+  export default content;
+}
```

Comment:
I'm not familiar with this kind of declaration, what does this achieve? Are these instructions for the build system?

Reply:
TypeScript doesn’t recognise the import otherwise, even though it’s correctly imported at runtime.

Reply:
So you transform the svg in string for typescript ?

Reply:
it is actually already a string when you do import upward from '../assets/images/ScheduledMaintenanceUp.svg';
looks like this
/@fs/home/ut/soft/Programmation/javascript/code/osrd-ui/ui-manchette-with-spacetimechart/src/assets/images/ScheduledMaintenanceUp.svg

Here it’s just to avoid getting typescrit errors